SOSS/dms/soss_centroids.py

```diff
@@ -0,0 +1,377 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import os
+import warnings
+
+import numpy as np
+
+from astropy.io import fits
+
+from .soss_utils import zero_roll, robust_polyfit, get_image_dim
+
+from matplotlib import colors
+import matplotlib.pyplot as plt
+
+PATH = '/home/talens-irex/Dropbox/SOSS_Ref_Files'
+
+
+def _plot_centroid(image, xtrace, ytrace):
+    """Overplot the extracted trace positions on the image."""
```

Comment:
Needs description of arguments "image", "xtrace", "ytrace"

Reply:
Added to docstring.